geometry_msgs/Pose pose # current position
float64 batteryLevel 
int8 robotId   
---
bool hasTask